spec/models/assignment_spec.rb

```diff
@@ -114,5 +114,78 @@
     assignment_node.node_object_id = assignment.id
     expect(assignment_node.belongs_to_course?).to be true
   end
+end
 
+describe "has correct csv values?" do
+  before(:each) do
+    @assignment = create(:assignment)
+    create(:assignment_team, name: "team1")
+    @student = create(:student, name: "student1")
+    create(:participant, user: @student)
+    create(:questionnaire)
+    create(:question)
+    create(:review_response_map)
+    create(:response)
+  end
+
+  delimiter = ","
+
+  it "checks_if_csv has the correct data" do
+    create(:answer, comments: "Test comment")
+    options = {"team_id" => "true", "team_name" => "true",
```

Comment:
You can put options in `before(:each)` and set the only change in each test case, which should remove the dup code. And after that I will approve other code climate issues, and there will be a green check for your pull request.